src/h5web/toolbar/controls/DomainSlider/ScaledSlider.tsx

```diff
@@ -78,8 +78,6 @@ function ScaledSlider(props: Props): ReactElement {
   return (
     <ReactSlider
       className={styles.slider}
-      thumbClassName={styles.thumb}
-      thumbActiveClassName={styles.thumbActive}
```

Comment:
- The thumb class name is now declared in the `Thumb` component.
- The thumb active class name was no longer used.